Classes/Issues/Files/IssuePatchContentViewController.swift

```diff
@@ -22,14 +111,79 @@ final class IssuePatchContentViewController: UIViewController {
 
         title = file.filename
 
+        let chunkContentSize = prepareChunkViews()
+
+        scrollView.contentSize = chunkContentSize
+    }
+
+    private func prepareChunkViews() -> CGSize {
         let width: CGFloat = 0
-        let text = NSAttributedStringSizing(
-            containerWidth: width,
-            attributedText: CreateDiffString(code: file.patch),
-            inset: Styles.Sizes.textViewInset
-        )
-        scrollView.contentSize = text.textViewSize(width)
-        attributeView.configureAndSizeToFit(text: text, width: width)
+        var chunkContentSize = CGSize.zero
+
+        let diffParser = DiffParser(content: self.file.patch)
+        guard diffParser.parse() == true else {
+            return chunkContentSize
+        }
+
+        // Iterate through all the chunks of this file and render it on the screen
+        diffParser.files.forEach { currentFile in
+            currentFile.chunks.forEach { chunk in
+                let chunkContent = "\(chunk.content)"
+
+                // Check the mode by combining the GitHub API data and what the diff parser detected
+                var fileMode = currentFile.mode
+                switch self.file.status {
+                case "added": fileMode = .added
+                case "deleted": fileMode = .deleted
+                default: fileMode = .modified
+                }
+
+                let chunkGutterRenderer = ChunkGutterRenderer(chunk: chunk, fileMode: fileMode)
+
+                let colorCodedChunkContent = CreateColorCodedString(code: chunkContent, includeDiff: true) { line, index in
+                    guard let lineInfo = chunkGutterRenderer.getLineDetails(for: index + 1) as LineMetadata? else { return .unknown }
+
+                    switch lineInfo.2 {
+                    case .addedLine: return .addedLine
+                    case .deletedLine: return .deletedLine
+                    default: return .unknown
+                    }
+                }
+
+                let text = NSAttributedStringSizing(
```

Comment:
A lot of these are default values, unnecessary to have them here

Reply:
Yes, I didn't want to change the order of the existing parameters :)

Reply:
We don't need to change the order of parameters, just remove the ones which are not the default implementation in this call to the init. Swift allows you to skip parameters where they have default values.